app/views/portfolio_items/_form.html.erb

```diff
@@ -1,31 +1,35 @@
-<!-- <div class="col-xs-12 col-md-6 col-md-offset-3">
-  <div class="form-container">
-    <%= form_for @portfolio_item remote: true do |f| %>
+<td colspan="6">
+  <div id="portfolio_itemForm">
```

Comment:
It is sort of weird to switch from snake case to camel case. You should stick with only one. CSS generally uses dashes.

Reply:
complete